modules/core/nuxeo-core-event/src/test/resources/test-rollback-listener.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<component name="test-rollback-listener">
```

Comment:
Maybe keep the usual convention in this module: `org.nuxeo.ecm.core.*`?

Reply:
I've followed the other files component name.